static/app/views/codecov/tests/tests.tsx

```diff
@@ -25,6 +26,7 @@ export default function TestsPage() {
         </PageFilterBar>
       </PageFiltersContainer>
       <Summaries />
+      <TestPreOnboardingPage />
```

Comment:
Should we add this to the /tests/new route from now? We'll eventually put everything together but feels like the right place to be